preview/MsixCore/Tests/notepadplus.msix

```diff
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:c93e7769fd5d028ff40bd7d14fbaebf92d30003eca0f8ea9b267aec9d1851d5f
-size 3112177
+oid sha256:070bb176c60c2e5165bccb5ae35301511150386611093592029837a667823d49
+size 3112179
```

Comment:
make sure this package works on win10 as well.

Reply:
test

---
In reply to: [316453431](https://github.com/microsoft/msix-packaging/pull/212#discussion_r316453431) [](ancestors = 316453431)